test/frontend/login/test_login.py

```diff
@@ -0,0 +1,25 @@
+from xml.dom import NotFoundErr
+from selenium import webdriver
+import selenium
+from selenium.webdriver.common.by import By
+
+import sys
+sys.path.append('../testlib')
+from register import *
+from teardown import *
+
+def test_login():
+    email = "test.account@gmail.com"
+    password = "TestAccount1@"
+    driver = login(email, password)
```

Comment:
Error: login not found.

Must be resolved by importing the **login** function from the login.py module. 

Please add "from login import *" before running test_login.